stestr/commands/run.py

```diff
@@ -134,8 +134,16 @@ def get_parser(self, prog_name):
         return parser
 
     def get_description(self):
-        help_str = (
-            "Run the tests for a project and store them into the repository.")
+        help_str = """Run the tests for a project and store them into the
+        repository.
```

Comment:
This change makes the `stestr help` output a bit odd.. (`list`'s description is also weird originally, though)
```
Commands:
  complete       print bash completion command (cliff)
  failing        Show the current failures known by the repository.
  help           print detailed help for another command (cliff)
  init           Create a new repository.
  last           Show the last run loaded into a repository.
  list           List the tests for a project. You can use a filter just like with the run command to see exactly what tests match
  load           Load a subunit stream into a repository.
  run            Run the tests for a project and store them into the
  slowest        Show the slowest tests from the last test run.
```

So, how about moving this to the class' docstring like this?[0]

```
class Run(command.Command):
    """Run the tests for a project and store them into the repository.

    Without --subunit, the process exit code will be non-zero if the test
    run was not successful. However, with --subunit, the process exit code
    is non-zero only if the subunit stream could not be generated
    successfully. The test results and run status are included in the
    subunit stream, so the stream should be used to determining the result
    of the run instead of the exit code when using the --subunit flag.
    """
```

> get_description()
> 
>     Return the command description.
> 
>     The default is to use the first line of the class’ docstring as the description. Set the _description class attribute to a one-line description of a command to use a different value. This is useful for enabling translations, for example, with _description set to a string wrapped with a gettext translation marker.
> 

[0] https://docs.openstack.org/cliff/latest/reference/index.html#cliff.command.Command.get_description

Reply:
Oops, I missed your comment here, let me try moving everything to a docstring locally and test that out. If it looks good I'll update it here, otherwise I think we should do it in a follow up (and I'll comment here).

Reply:
Yep, the docstrings are the right way to handle this. I've updated things accordingly in https://github.com/mtreinish/stestr/pull/214/commits/fef78ac369298b602fd18cef6c68550fefd93a41